sdks/python/apache_beam/metrics/cells.py

```diff
@@ -77,6 +79,13 @@ def reset(self):
     # type: () -> None
     raise NotImplementedError
 
+  @property
+  def start_time(self):
```

Comment:
Could simplify this a bit by putting the start time logic in MetricCell.to_runner_api_monitoring_info and having that call a to_runner_api_monitoring_info_impl on each of the subclasses

Reply:
Something like this:

class MetricCell:
  def to_runner_api_monitoring_info(self, name, transform_id):
    mi = self.to_runner_api_monitoring_info_impl(name, transform_id)
    mi.start_time.FromDatetime(self.start_time) # or just put the start_time code inline here
    return mi

Then, rename all the to_runner_api_monitoring_info methods in the subclasses to to_runner_api_monitoring_info_impl